lmdeploy/serve/gradio/vl.py

```diff
@@ -174,7 +174,10 @@ def cancel(chatbot, session):
 
     def reset(session):
         """Reset a new session."""
-        stop(session)
+        if session is None:
```

Comment:
Why is it a None?

Reply:
press `reset` button before input anything, the `session is None`